ISL.Reidentification.Core/Models/Foundations/AccessAudit/AccessAudit.cs

```diff
@@ -0,0 +1,20 @@
+﻿// ---------------------------------------------------------
+// Copyright (c) North East London ICB. All rights reserved.
+// ---------------------------------------------------------
+
+using System;
+
+namespace ISL.Reidentification.Core.Models.Foundations.AccessAudit
+{
+    public class AccessAudit
```

Comment:
You need the latest code from main to apply this

```suggestion
    public class AccessAudit : IKey, Iaudit
```